app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -929,12 +933,45 @@ export default Controller.extend({
     // --> pop the window up
     if (!force && composer.replyingToTopic) {
       const currentTopic = this.topicModel;
+      const originalTopic = this.model.topic;
 
       if (!currentTopic) {
         this.save(true);
         return;
       }
 
+      let topicLabelContent = function (topicOption) {
```

Comment:
```suggestion
      const topicLabelContent = function (topicOption) {
```

I believe this isn't being edited anywhere.